contrib/build-push-ceph-container-imgs.sh

```diff
@@ -201,7 +201,7 @@ function create_registry_manifest {
   # This should normally work, by the time we get here the arm64 image should have been built and pushed
   # IIRC docker manisfest will fail if the image does not exist
   for image in daemon-base daemon; do
-    for ceph_release in luminous mimic; do
+    for ceph_release in ${CEPH_RELEASES[@]:1}; do
```

Comment:
I'm doubting now, if we call `contrib/build-push-ceph-container-imgs-arm64.sh ` that's going to override`CEPH_RELEASES` with the one in `contrib/build-push-ceph-container-imgs-arm64.sh` so removing the first element will remove `luminous` in this case right?

Reply:
I'll take a closer look but I think you're right so we probably need to set the list explicitly and not reuse the CEPH_RELEASES variable

Reply:
in fact this has no side effect for arm64 because the `create_registry_manifest` function is only executed from `contrib/build-push-ceph-container-imgs.sh` (so with the right list).
With the arm64 script the function does nothing. https://github.com/ceph/ceph-container/blob/master/contrib/build-push-ceph-container-imgs-arm64.sh#L45-L47